moveit_cartesian_plan_plugin/src/generate_cartesian_path.cpp

```diff
@@ -128,40 +181,55 @@ void GenerateCartesianPath::checkWayPointValidity(const geometry_msgs::Pose& way
       /*! This function is called every time the user updates the pose of the Way-Point and checks if the Way-Point is within the valid IK solution for the Robot.
           In the case when a point is outside the valid IK solution this function send a signal to the RViz enviroment to update the color of the Way-Point.
       */
-       bool found_ik = kinematic_state->setFromIK(joint_model_group, waypoint, 3, 0.005);
+       bool found_ik = kinematic_state->setFromIK(joint_model_group, waypoint, 3, 0.006);
 
          if(found_ik)
         {
 
-           Q_EMIT wayPointOutOfIK(point_number,0); 
+           Q_EMIT wayPointOutOfIK(point_number,0);
         }
         else
         {
-          ROS_INFO("Did not find IK solution for waypoint %d",point_number);
-          Q_EMIT wayPointOutOfIK(point_number,1); 
+          // ROS_INFO("Did not find IK solution for waypoint %d",point_number);
+          Q_EMIT wayPointOutOfIK(point_number,1);
         }
 }
-// //doesnt make sense to put it in concurent process, the update is not realistic
-// void GenerateCartesianPath::checkWayPointValidityHandler(const geometry_msgs::Pose& waypoint,const int point_number)
-// {
-//     ROS_INFO("Concurrent process for the Point out of IK range");
-//     QFuture<void> future = QtConcurrent::run(this, &GenerateCartesianPath::checkWayPointValidity, waypoint,point_number);
-
-// }
 
 void GenerateCartesianPath::initRviz_done()
```

Comment:
Doesn't follow naming convention.  Is there a reason?  This is not a new change, so feel free to leave it.